src/Graphics/Canvas.js

```diff
@@ -53,20 +53,20 @@ exports.getCanvasHeight = function(canvas) {
     };
 };
 
-exports.setCanvasWidth = function(width) {
-    return function(canvas) {
+exports.setCanvasWidth = function(canvas) {
+    return function(width) {
         return function() {
             canvas.width = width;
-            return canvas;
+            return {};
```

Comment:
For reference, you don't need these, since values of type `Unit` can't be inspected, so you can return any value and the user won't be able to observe it.

There could be a small speedup to be had by not constructing these values.

Reply:
sure. that's easy to change. just pushed an update